CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.3.5 - 2025-07-31
+
+### Added
+- Simplify project compilation with .props file
+
+### Fixed
+- Use correct zeiss SDK character (µm) to fix encoding issue
```

Comment:
Just a note: Since it was working with 0.3.4, this is not really something that got fixed by 0.3.5.